Source/WebKit/WebProcess/Plugins/PluginView.h

```diff
@@ -169,6 +169,8 @@ class PluginView final : public WebCore::PluginViewBase {
 
     bool pluginDelegatesScrollingToMainFrame() const;
 
+    bool presentingLockedContent() const;
```

Comment:
[WebKit coding guidelines](https://webkit.org/code-style-guidelines/#names-bool) recommend bool values (and I thought function names too, but the guidelines don't mention that) start with "is" / "has" / "did" / etc. So maybe `is PresentingLockedContent()`?

Reply:
Good point, I will prefix with `is`